docs/learning/training/index.md

```diff
@@ -5,8 +5,6 @@ technical development.
 
 A list of training can be seen below:
 
-## Training
-
 - Installing npm
 - Product Epics
 - User Stories
```

Comment:
Should these be linked? and would it be better to just link to the training folder?

Reply:
Thanks, @talzid. From what I understand, we do not have these resources built yet. I could not find them anywhere in the handbook repo 🤔

Reply:
Good point. Might be worth linking npm to the contribution file, user stories to the QA section, and removing product epics?

> would it be better to just link to the training folder? 

My mistake didn't realise that's where this was

Reply:
> user stories to the QA section, and removing product epics?

Good idea. I will link epics and user stories to the QA section

> npm to the contribution file

Will drop the NPM installation for now. I will bring the one Morgaine created in the documentation repo over here.